website/database.py

```diff
@@ -149,6 +149,9 @@ def get_token(self, token_id):
 
     def store_token(self, token):
         """Store a token in the database."""
+
+        # But first, remove all already existing tokens for this user
+        TOKENS.del_many({'username': token['username']})
```

Comment:
I think if we use the expiry date, this is no longer needed?

Reply:
I will have to take a look at this all together! Will have to make some changes so the reset tokens are handled differently than the login tokens. Will put it into a draft until I figured this out.

Reply:
When look at this we can indeed definitely remove this! The removal should happen on the database-side. The only moment this should be called is after the password reset. Please help me remember to index the usernames on the database (and turn on the TTL setting) in Dynamo before we merge this.